### PR TITLE
Price range changes.

### PR DESCRIPTION
**Pull request by ChaitanyaV98**

### CODE REVIEW

File: app/requests/createGmProductsConstraints.js [hunk 1]
```diff
@@ -84,6 +84,37 @@ validate.validators.categoriesValidator = function (value) {
   });
 };
 
+validate.validators.maxPriceValidator = function (value, flag, key, data) {
+  return new validate.Promise(function (resolve, reject) {
+    if (value) {
+      // value should be in number format
+      const maxValueRegex = /^[1-9]\d*(\.\d+)?$/;
```
Comment (ShreyDhyani) [inline app/requests/createGmProductsConstraints.js:91]: @Subbu-betalectic 0.9, 0.8 are invalid here

File: app/requests/createGmProductsConstraints.js [hunk 2]
```diff
@@ -84,6 +84,37 @@ validate.validators.categoriesValidator = function (value) {
...
+      resolve();
+    }
+  });
+};
+
+validate.validators.numberValidator = function (value) {
+  return new validate.Promise(function (resolve, reject) {
+    if (value) {
+      // value should be in number format
+      const numberRegex = /^[1-9]\d*(\.\d+)?$/;
```
Comment (ShreyDhyani) [inline app/requests/createGmProductsConstraints.js:108]: @Subbu-betalectic here too

File: app/controllers/gmBulkProduct.js [hunk 3]
```diff
@@ -279,7 +279,7 @@ module.exports = {
           }
 
           // Validate product method will returns either empty object or object with error keys
-          const validationResult = await validateProduct(product, constraints);
+          const validationResult = await validateProduct(product);
```
Comment (ShreyDhyani) [inline app/controllers/gmBulkProduct.js:282]: Not checking Against constraints now??

Comment (ShreyDhyani) [reply app/controllers/gmBulkProduct.js:282]: @Subbu-betalectic

Comment (Subbu-betalectic) [reply app/controllers/gmBulkProduct.js:282]: @ShreyDhyani Removed the unnecessary parameter

**Pull request merged**